packages/web-react/src/types/textArea.ts

```diff
@@ -24,6 +24,8 @@ export interface TextAreaProps
   isAutoResizing?: boolean;
   /** Maximum field height with automatic height control */
   autoResizingMaxHeight?: number;
+  /** @deprecated The "id" property will be required instead of optional starting from the next major version. */
```

Comment:
we should trigger useDeprecationMessage.ts hook if the id is not set inside these components.

Reply:
added deprecation hook to all components 👍